### PR TITLE
ci: Add E2E tests for Webhook node

### PR DESCRIPTION
**Pull request by flipswitchingmonkey**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: cypress/e2e/16-webhook-node.cy.ts [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+		cy.wait(waitForWebhook);
+
+		cy.request(method, '/webhook-test/'+ webhookPath).then((response) => {
+			expect(response.status).to.eq(200);
+			ndv.getters.outputPanel().contains('headers');
+		});
+	}
+};
+
+describe('Schedule Trigger node', async () => {
```
Comment (mutdmour) [inline cypress/e2e/16-webhook-node.cy.ts:100]: I guess this needs to be renamed

File: cypress/e2e/16-webhook-node.cy.ts [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+
+	it('should listen for a GET request', ()=>{
+		simpleWebhookCall({method: 'GET', webhookPath: uuid(), executeNow: true});
+	});
+
+	it('should listen for a POST request', ()=>{
+		simpleWebhookCall({method: 'POST', webhookPath: uuid(), executeNow: true});
+	});
+
+	it('should listen for a DELETE request', ()=>{
```
Comment (mutdmour) [inline cypress/e2e/16-webhook-node.cy.ts:119]: nitpick - spacing
```suggestion
	it('should listen for a DELETE request', () => {
```

**Review by mutdmour — APPROVED** [review]
nice job

**Comment by janober** [comment]
Got released with `n8n@0.215.0`

**Pull request merged**